src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -558,10 +558,90 @@
     <value>These symbols will be used when you double-click text in the terminal or activate "Mark mode".</value>
     <comment>A description for what the "word delimiters" setting does. Presented near "Globals_WordDelimiters.Header". "Mark" is used in the sense of "choosing something to interact with."</comment>
   </data>
+  <data name="AISettings_ServiceProvidersHeader.Text" xml:space="preserve">
+    <value>Service Providers</value>
+    <comment>Header for a group of settings related to the AI service providers.</comment>
+  </data>
+  <data name="AISettings_Disclaimer1.Text" xml:space="preserve">
+    <value>An AI key and endpoint are required for accessing AI services within Windows Terminal. Credentials are securely stored by</value>
+    <comment>Header for the AI settings page, where the user stores their AI key and endpoint for AI services within Windows Terminal.</comment>
+  </data>
+  <data name="AISettings_DisclaimerLink" xml:space="preserve">
+    <value>Windows Credential Manager</value>
+    <comment>Header for the AI settings page, where the user stores their AI key and endpoint for AI services within Windows Terminal.</comment>
+  </data>
+  <data name="AISettings_Disclaimer2.Text" xml:space="preserve">
+    <value>and not in the JSON file.</value>
+    <comment>Header for the AI settings page, where the user stores their AI key and endpoint for AI services within Windows Terminal.</comment>
+  </data>

```

Comment:
another one of them sentences that's broken across resources

Reply:
(et al in this file)

Reply:
Going to resolve this convo here to keep the discussion about localization in the same place (in the comment thread on the other resource file)